Magento_GiftCardAccount/web/template/summary/gift-card-account.html

```diff
@@ -0,0 +1,28 @@
+<!-- ko if: isAvailable() -->
+    <!-- ko foreach: getAppliedGiftCards() -->
+    <li class="order-summary__list-item">
+        <div>
+            <span class="order-summary__label">
+                <!-- ko text: $parent.title + ' (' + c + ')' --><!-- /ko -->
+            </span>
+            <div>
+                <a
```

Comment:
Could you check if you can easly change it to `button`?

Reply:
`button--link` class

Reply:
I'll add `button--link` class. But it is `<a>` tag https://magento2-alpaca-theme-git-master-snowdog1.vercel.app/components/detail/button--link.html